script/app.js

```diff
@@ -60,17 +59,45 @@ class BookSet {
     localStorage.setItem('bookItems', JSON.stringify({ bookColl: this.books }));
   }
 }
-
 // Book Constructor function (representing a book).
-
 const coll = new BookSet();
 if (localStorage.getItem('bookItems')) {
   const localBooks = JSON.parse(localStorage.getItem('bookItems'));
   localBooks.bookColl.forEach((item) => {
     coll.add(new Book(item.title, item.author));
   });
 }
-
 submitBtn.addEventListener('click', () => {
   coll.add(new Book(inputTitle.value, inputAuthor.value));
 });
+
+// Navigation
+
+const dateVisit = document.querySelector('.userDate');
```

Comment:
- [ ] [OPTIONAL] Great job so far. Your web app looks great 👏. However, I observed some unexpected behavior (bug) in your app which would be better if you work on it. See when I add a book it is displayed in the book list section and it is also stored in the local storage. **However, when I reload/refresh the page, there is nothing that appears in the book list section while they are supposed to appear books stored in the local storage**. The book appears again only when I add another one then, they appear as two. **This may be because there is no function that checks if there are books in the local storage and displays them every time on initial page loading**. Please add that function 😃 👍

Reply:
thank you for the Feedback @BertrandConxy i'll work on this